_episodes/01-remotes.md

```diff
@@ -201,8 +201,14 @@ It will work for one more person.
 
 ### 10. Create a branch `yourname/somefeature` pointing at your commit
 
-First find out the hash of your commit (using `git graph` or `git
-log`), then create a branch "in the past" pointing to that hash:
+First find out the hash of your commit. You can do this using `git graph` or `git log`
+but another nice way of finding your commit is to use `git blame` on the file which you have added:
+
+```
+$ git blame yourusername.txt
```

Comment:
```suggestion
$ git log yourusername.txt
```
See comment in #83 for why I suggest this.  The paragraph above would have to be adjusted too.

Reply:
Also then https://github.com/coderefinery/template-centralized-workflow-exercise/pull/1 needs updating.

Reply:
I agree. Working on adjusting this ...

Reply:
Done, both here and in the exercise repo.